app/models/annotations/status.rb

```diff
@@ -105,6 +105,10 @@ def destroy
 
   private
 
+  def set_annotator
+    self.annotator = User.current unless User.current.nil?
+  end
+
```

Comment:
We already have this method here: https://github.com/meedan/check-api/blob/develop/app/models/concerns/annotation_base.rb#L266. Why we need it here too?